src/main/java/de/tum/in/www1/artemis/repository/ModelingExerciseRepository.java

```diff
@@ -34,8 +34,23 @@ public interface ModelingExerciseRepository extends JpaRepository<ModelingExerci
     @EntityGraph(type = LOAD, attributePaths = { "exampleSubmissions", "teamAssignmentConfig", "categories", "competencies", "exampleSubmissions.submission.results" })
     Optional<ModelingExercise> findWithEagerExampleSubmissionsAndCompetenciesById(@Param("exerciseId") Long exerciseId);
 
-    @Query("select modelingExercise from ModelingExercise modelingExercise left join fetch modelingExercise.exampleSubmissions exampleSubmissions left join fetch exampleSubmissions.submission submission left join fetch submission.results results left join fetch results.feedbacks left join fetch results.assessor left join fetch modelingExercise.teamAssignmentConfig where modelingExercise.id = :#{#exerciseId}")
-    Optional<ModelingExercise> findByIdWithExampleSubmissionsAndResults(@Param("exerciseId") Long exerciseId);
+    @EntityGraph(type = LOAD, attributePaths = { "exampleSubmissions", "teamAssignmentConfig", "categories", "competencies", "exampleSubmissions.submission.results",
+            "plagiarismDetectionConfig" })
+    Optional<ModelingExercise> findWithEagerExampleSubmissionsAndCompetenciesAndPlagiarismDetectionConfigById(@Param("exerciseId") Long exerciseId);
+
+    @Query("""
+               SELECT modelingExercise\s
+               FROM ModelingExercise modelingExercise\s
+               LEFT JOIN FETCH modelingExercise.exampleSubmissions exampleSubmissions\s
+               LEFT JOIN FETCH exampleSubmissions.submission submission\s
+               LEFT JOIN FETCH submission.results results\s
+               LEFT JOIN FETCH results.feedbacks\s
+               LEFT JOIN FETCH results.assessor\s
+               LEFT JOIN FETCH modelingExercise.teamAssignmentConfig\s
+               LEFT JOIN FETCH modelingExercise.plagiarismDetectionConfig\s
+               WHERE modelingExercise.id = :#{#exerciseId}
+            """)
```

Comment:
Why do you use `\s` here? Also, please use our common SQL style (`:exerciseId`, extra indentation for `LEFT JOIN FETCH`)

Reply:
Done